### PR TITLE
Support default option in SelectBox

### PR DESCRIPTION
**Pull request by rszwajko**
Mark default options on Account Settings screen for:
1. language
default_language
2. refresh interval
default_referesh_interval

### CODE REVIEW

File: src/components/SelectBox.js [hunk 1]
```diff
@@ -60,10 +60,13 @@ class SelectBox extends React.Component {
   }
 
   render () {
-    const { id, disabled } = this.props
+    const { id, disabled, msg } = this.props
 
     const selectedItem = this.state.items.find(item => item.id === this.state.selected)
     const validationClass = this.getValidationClass()
+    const markAsDefault = () => {
```
Comment (sjd78) [inline src/components/SelectBox.js:67]: I don't think this needs to be a function....

You should be able to do:
```jsx
const MarkAsDefault = <i>{msg.defaultOption()}</i>
```

and then later just do:
```jsx
  {selectedItem && selecteItem.default && <MarkAsDefault />}
```

**Review by sjd78 — CHANGES_REQUESTED** [review]
Just a suggestion on the code style in `SelectBox`

**Review by hstastna — APPROVED** [review]
LGTM :)

**Comment by rszwajko** [comment]
@sjd78 
Changes:
1. as suggested moved more responsibility to `MarkAsDefault` component
2. renamed the flag to`isDefault` to prevent name clash with `default` as reserved word

**Review by sjd78 — APPROVED** [review]
LGTM

**Pull request merged**